client/components/purchase-detail/index.jsx

```diff
@@ -70,6 +70,7 @@ PurchaseDetail.propTypes = {
 	description: React.PropTypes.oneOfType( [
 		React.PropTypes.array,
 		React.PropTypes.string,
+		React.PropTypes.object
```

Comment:
So I guess it can be anything? Would be cool to add a custom propTypes for objects returned by `this.translate` (not for this PR I mean)